cloud: update ui description for org settings (#16348)
pingcap/docs: tidb-cloud/manage-user-access.md

2. Click **Organization Settings**. The organization settings page is displayed.

> 3. In the **Time Zone** section, click the drop-down list and select your time zone.

Reviewer: ```suggestion
3. In the **Time Zone** section, select your time zone from the drop-down list.
```
